src/terms/cache-type.md

```diff
@@ -8,4 +8,4 @@
   relatedTerms:
     - "cache-types"
 ---
-A cache stores data so that future calls for that data can be loaded quicker, and Magento includes these types: configuration, layout, block HTML layout, full page (the most well-known), collections, DDL, EAV, reflection, translation, integration configuration, integration API configuration, web services configuration. Other types can be created and defined.
+A cache stores data so that future calls for that data can be loaded quicker.
```

Comment:
I think it would be better to add the content you created in the new `cache-types.md` file here instead of creating a new term and just remove the `relatedTerms` metadata from this file

Reply:
Hello @jcalcaben 
Thank you for review.
I have fixed it.
Please check.